pages/narratives/_book/_chapter/_page.vue

```diff
@@ -34,6 +70,6 @@ export default {
   margin-top: 62.5vh;
 }
```

Comment:
You can add a [methods](https://vuejs.org/v2/guide/components.html#Form-Input-Components-using-Custom-Events) section here.